src/options.rs

```diff
@@ -433,7 +433,7 @@ pub fn build_app(
         use_cpu: used_widget_set.get(&Cpu).is_some() || used_widget_set.get(&BasicCpu).is_some(),
         use_mem,
         use_cache: use_mem && get_enable_cache_memory(matches, config),
-        use_gpu: use_mem && get_enable_gpu_memory(matches, config),
+        use_gpu: use_mem && get_enable_gpu(matches, config),
```

Comment:
Can probably just do:

```suggestion
        use_gpu: get_enable_gpu(matches, config),
```